addon/utils/get-position.js

```diff
@@ -35,3 +36,12 @@ export default function getPosition(el) {
 
   return assign({}, clonedRect, scroll, outerDims, elOffset);
 }
+
+
+function getScrollPosition() {
+  return {
+    //    IE, Firefox                            Chrome
+    top:  document.documentElement.scrollTop  || document.body.scrollTop  || 0,
```

Comment:
I was checking which browsers are targeted by ```window.page[XY]Offset``` but couldn't find a clear answer. As most seem to work with the above two, I added them for now. Tested in FF, Safari, Chrome